transformer/app.js

```diff
@@ -4,9 +4,11 @@ import { DynamoDBClient, QueryCommand } from '@aws-sdk/client-dynamodb';
 import { getLogger } from './logger.js';
 
 const logger = getLogger();
+const REGION = process.env.AWS_REGION;
+const SECURITY_HUB_REGION = process.env.SECURITY_HUB_REGION || REGION;
 
 const ddbClient = new DynamoDBClient();
-const hubClient = new SecurityHubClient();
+const hubClient = new SecurityHubClient({ region: SECURITY_HUB_REGION });
```

Comment:
I wonder if this is necessary. Probably don't do any harm.

Reply:
It is necessary. https://jira.jfrog.org/browse/PTRENG-4401

Reply:
That ticket, the way I read it from Scott's comment during the meeting, is that the region needs to be part of the integration ARN. I don't know if this region needs to be set for the SDK client.